doc/pages/intro_install.rst

```diff
@@ -1,12 +1,12 @@
 .. highlight:: bash
 
 Welcome to the libnrm guide. This document will help you in your use of the
-C/C++ and Fortran interface for NRM.
+C/C++ interface for NRM.
```

Comment:
ok my bad, it looks like the previous version was C++ proof, not the case right now.